test/wampy-serializer-handshake-test.js

```diff
@@ -1,60 +1,49 @@
-
-/**
- * Project: wampy.js
- * User: KSDaemon
- * Date: 13.06.17
- */
 import { expect } from 'chai';
 import { WebSocket, setProtocol as wsSetProtocol } from './fake-ws-set-protocol.js';
 import { Wampy } from './../src/wampy.js';
 import { JsonSerializer } from '../src/serializers/JsonSerializer.js';
 import { CborSerializer } from '../src/serializers/CborSerializer.js';
-import * as Errors from '../src/errors.js';
-
-const routerUrl = 'ws://fake.server.org/ws/';
+import { NoSerializerAvailableError } from '../src/errors.js';
 
 describe('Wampy.js Serializer Handshake', function () {
     this.timeout(0);
 
-    it('disallows to connect when server choose not available serializer', async function () {
-        let wampy = new Wampy(routerUrl, {
-            realm: 'AppRealm',
-            ws: WebSocket,
-            serializer: new JsonSerializer()
-        });
-        wsSetProtocol('cbor');
+    const testUrl = 'ws://fake.server.org/ws/';
+    const defaultOptions = { realm: 'AppRealm', ws: WebSocket };
+    const jsonSerializerOptions = { ...defaultOptions, serializer: new JsonSerializer() };
+    const cborSerializerOptions = { ...defaultOptions, serializer: new CborSerializer() };
 
+    it('disallows to connect when server choose not available serializer', async () => {
         try {
+            const wampy = new Wampy(testUrl, jsonSerializerOptions);
+
+            wsSetProtocol('cbor');
             await wampy.connect();
-        } catch (e) {
-            expect(e).to.be.instanceOf(Errors.NoSerializerAvailableError);
+        } catch (error) {
```

Comment:
Do you want to rename all e→error everywhere across tests?) It's more than 100 :)
I think it's a pretty standard construction for one liners (small) catch blocks. Let's keep it?

Reply:
I agree!